labelme/app.py

```diff
@@ -949,16 +959,16 @@ def main():
     """Standard boilerplate Qt application code."""
     parser = argparse.ArgumentParser()
     parser.add_argument('filename', nargs='?', help='image or label filename')
-    parser.add_argument('-O', '--output', help='output label name')
+    parser.add_argument('--output', '-O', '-o', help='output label name')
+    parser.add_argument('--nodata', '-nd', dest='store_data',
```

Comment:
The short flag should only be one character, otherwise it looks like you'd call it with `-n` and `-d`

Reply:
Actually I wouldn't add a short flag at all. It makes shell scripts unreadable and normally you'd recall it from your shell history anyways, so no time saved...

Reply:
Thanks! I updated my commit.